aws-codeguruprofiler-profilinggroup/aws-codeguruprofiler-profilinggroup.json

```diff
@@ -16,6 +20,27 @@
       "maxLength": 255,
       "pattern": "^[\\w-]+$"
     },
+    "Permissions": {
+      "description": "The permissions attached for this profiling group.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "AgentPermissions": {
+          "type": "object",
+          "description": "The permissions for the agent.",
+          "additionalProperties": false,
+          "properties": {
+            "Principals": {
+              "description": "The principals for the agent permissions.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ArnIam"
+              }
+            }
+          }
+        }
+      }
+    },
```

Comment:
just fyi, the schema changes are updated and reviewed in [PR #10](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-codeguru-profiler/pull/10).